src/assets/icons/sad-emoji.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This XML declaration isn't necessary. Are you using Illustrator and the "export for screens" dialog?